.github/workflows/build.yml

```diff
@@ -169,7 +169,8 @@ jobs:
             const commentBody = `${ TEST_COMMENT_INDICATOR }
             Are you an Automattician? Please test your changes on all WordPress.com environments to help mitigate accidental explosions.
 
-            - To test on WoA, go to the Plugins menu on a WordPress.com Simple site. Click on the "Upload" button and follow the upgrade flow to be able to upload, install, and activate [the Jetpack Beta plugin](https://jetpack.com/download-jetpack-beta/). Once the plugin is active, go to Jetpack > Jetpack Beta > Jetpack and enable the \`${ BRANCH_NAME }\` branch.
+            - To test on WoA, go to the Plugins menu on a WordPress.com Simple site. Click on the "Upload" button and follow the upgrade flow to be able to upload, install, and activate [the Jetpack Beta plugin](https://jetpack.com/download-jetpack-beta/). Once the plugin is active, go to Jetpack > Jetpack Beta > Jetpack and enable the \`${ BRANCH_NAME }\` branch. 
+            - If you are testing a \`jetpack-mu-wpcom\` change you will need to go to Jetpack > Jetpack Beta > WordPress.com Features and enable the \`${ BRANCH_NAME }\` branch. You will also need to add \`define( 'JETPACK_MU_WPCOM_LOAD_VIA_BETA_PLUGIN', true );\` to your \`wp-config.php\` file on your WoA dev blog.
```

Comment:
This seems a bit repetitive.
```suggestion
            - To test on WoA, go to the Plugins menu on a WordPress.com Simple site. Click on the "Upload" button and follow the upgrade flow to be able to upload, install, and activate [the Jetpack Beta plugin](https://jetpack.com/download-jetpack-beta/). Once the plugin is active, go to Jetpack > Jetpack Beta, select your plugin, and enable the \`${ BRANCH_NAME }\` branch. 
              - For \`jetpack-mu-wpcom\` changes, also add \`define( 'JETPACK_MU_WPCOM_LOAD_VIA_BETA_PLUGIN', true );\` to your \`wp-config.php\` file.
```
Then even better would be to skip the extra bit about jetpack-mu-wpcom if `data.projects` doesn't contain `jetpack-mu-wpcom-plugin`.

Reply:
I updated the PR with the suggested changes, but now I am seeing two recommended commands in the comments for this PR, which seems wrong AND not related to this change. Do you know why that might be?
This is what I am referring to: 
<img width="809" alt="image" src="https://github.com/Automattic/jetpack/assets/375980/b7472b24-2358-4b84-a9e7-875e2b82e4b9">

Reply:
There's one command for Jetpack and one for the jetpack-mu-wpcom-plugin. Since the PR touches build.yml, [everything gets built](https://github.com/Automattic/jetpack/actions/runs/6316528518/job/17151350443#step:6:16) to give the change a proper test.

I saw that before your recent change to the PR too.